### PR TITLE
feat(electron): Add `BrowserWindowSession` integration

### PR DESCRIPTION
**Pull request by timfish**
Add docs for the new `BrowserWindowSession` integration

### CODE REVIEW

File: src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: BrowserWindowSession
+excerpt: ""
+description: "Captures sessions linked to the focus of Electron BrowserWindows."
+sidebar_order: 15
+---
+
```
Comment (AbhiPrasad) [inline src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx:7]: Let's add a note that this requires `4.11.0`.

Should we also talk about this affects the MainProcessSession integration?

File: src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Requires `@sentry/electron` version `4.11.0` or higher.
+
+Captures sessions linked to the focus of Electron `BrowserWindow`'s. Capturing
+sessions makes it possible to show [release health](/product/releases/health/)
+statistics in Sentry.
+
+When this integration has been added, the `MainProcessSession` session
```
Comment (AbhiPrasad) [inline src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx:14]: can we add a link to the other integration docs here?

**Review by AbhiPrasad — APPROVED** [review]

File: src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+excerpt: ""
+description: "Captures sessions linked to the focus of Electron BrowserWindows."
+sidebar_order: 15
+---
+
+Requires `@sentry/electron` version `4.11.0` or higher.
+
+Captures sessions linked to the focus of Electron `BrowserWindow`'s. Capturing
+sessions makes it possible to show [release health](/product/releases/health/)
+statistics in Sentry.
```
Comment (shanamatthews) [inline src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx:12]:
```suggestion
Captures sessions linked to the focus of Electron `BrowserWindow`s. Capturing
sessions makes it possible to show [release health](/product/releases/health/)
statistics in Sentry.
```

File: src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+When this integration has been added, the <PlatformLink to="/configuration/integrations/mainprocesssession">MainProcessSession</PlatformLink>
+session integration will be disabled.
```
Comment (shanamatthews) [inline src/platforms/javascript/guides/electron/configuration/integrations/browserwindowsession.mdx:15]:
```suggestion
Adding this integration disables the <PlatformLink to="/configuration/integrations/mainprocesssession">MainProcessSession</PlatformLink>
session integration.
```

**Review by shanamatthews — APPROVED** [review]
Just a couple language tweaks, otherwise looks good!

**Pull request merged**